storage/src/cache/filecache/cache_entry.rs

```diff
@@ -555,6 +557,8 @@ impl FileCacheEntry {
                         req.tags[i].clone(),
                         None,
                     )?;
+                } else {
+                    state.commit()
```

Comment:
What does `commit` mean? The only place a merge state is committed, is when there are non-user IO in the fast cache case. The committed status is for the entire merge state, but it only matters in the above-dropped non-user IO case.

Reply:
I should add comment to describe the newly introduced method `commit` to close the understanding gap.
The *MergeState* status is only for last region within the merge state since it is hard for `MergeState` alone to tell if a chunk is skipped to be pushed into, although the chunks in merge requests(req) must be continuous. So we **Must** give the merge state extra hint.

Because `push` method of `MergeState` checks if input chunk is continuous with the last one in regions.